### ISSUE TITLE
Unable to use Steam Controller Configurations in Oblivion

### ISSUE THREAD

**Issue by Zetawilk** [ModOrganizer2/modorganizer#1257]
## The problem:
Unable to use Steam Controller Configurations in Oblivion. 

## To Reproduce:
Open Oblivion as instructed by this Github. 

## Environment:
MO2 v2.3.2
Windows 10

## Details:
Unable to use Steam Controller Configurations in Oblivion.

**Comment by LostDragonist** [comment]
Yeah, that's a thing that happens for some people.  We don't really know why and don't have much control over it.  One thing you can try is adding MO to Steam as a non-Steam game and running it from there.  Though I'm sure that messes up something else.
